src/common/InputWithClearButton.js

```diff
@@ -14,10 +14,10 @@ const componentStyles = StyleSheet.create({
     justifyContent: 'center',
     width: 22,
     height: 22,
-    marginRight: 12,
   },
   clearButtonIcon: {
     color: BRAND_COLOR,
+    marginRight: 8,
```

Comment:
When I look at this change, the first thing I am wondering is: Why is this value `8` if we had `12` before. Maybe there is a good reason but would be good if you mention it.

Reply:
Even better though, since this is an element that is `touchable` it needs to be as big as possible. If you use `padding` instead of `margin` the touch area will increase, because the `padding` space is part of the component. A possible solution is to set the `height` and `width` to `22` of the `clearButtonInput` and don't use `padding`.